### ISSUE TITLE
Not working on Gnome 40

### ISSUE THREAD

**Issue by Truemmerer** [iamlemec/miniview#24]
I have added the following in the metadata.json under shell-version: "40",
This seems to work for me under both Xorg and Wayland.

Fedora 34

**Comment by iamlemec** [comment]
Yup, thanks! Seeing the same thing here. I'll add in "40.alpha" and "40.beta" as well just to be sure. Still working on getting settings working with the switch to GTK4.
